x-pack/plugins/security_solution/public/common/hooks/use_providers_portal.tsx

```diff
@@ -6,13 +6,13 @@
  */
 
 import { useState } from 'react';
-import { createPortalNode } from 'react-reverse-portal';
+import { createHtmlPortalNode } from 'react-reverse-portal';
 
 /**
  * A singleton portal for rendering the draggable groups of providers in the
  * header of the timeline, or in the animated flyout
  */
-const proivdersPortalNodeSingleton = createPortalNode();
+const proivdersPortalNodeSingleton = createHtmlPortalNode();
 
 export const useProvidersPortal = () => {
   const [proivdersPortalNode] = useState(proivdersPortalNodeSingleton);
```

Comment:
```suggestion
  const [providersPortalNode] = useState(providersPortalNodeSingleton);
```

Reply:
and there is another use at 20th line, for which I for some reason can't add a suggestion